dces-drc-integration/build.gradle

```diff
@@ -127,7 +129,16 @@ dependencies {
 	testImplementation "com.squareup.okhttp3:mockwebserver:$versions.mockwebserverVersion"
 
 	testCompileOnly 'org.projectlombok:lombok'
+	testCompileOnly 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
 	testAnnotationProcessor 'org.projectlombok:lombok'
+
+	testImplementation "net.serenity-bdd:serenity-junit5:$versions.serenityVersion"
+	testImplementation "net.serenity-bdd:serenity-core:$versions.serenityVersion"
+	testImplementation "net.serenity-bdd:serenity-rest-assured:$versions.serenityVersion"
+
+	testImplementation "org.apache.commons:commons-configuration2:2.10.1"
+	testImplementation "commons-beanutils:commons-beanutils:1.9.4"
```

Comment:
Are you sure we need either of these?

- I don't see commons-beanutils being used (and Spring has its own `BeanUtils` class that does 99% of what it does)
- The usage of commons-configuration looks like a straightforward `java.util.Properties` would do